src/agent.rs

```diff
@@ -143,9 +147,17 @@ impl Agent {
             self.config.pythd_api_server.clone(),
             pythd_adapter_tx,
             shutdown_rx,
-            logger,
+            logger.clone(),
         ));
 
+        // Spawn the metrics server
+        jhs.push(tokio::spawn(metrics::MetricsServer::spawn(
+            "127.0.0.1:8888".parse::<SocketAddr>()?,
```

Comment:
I think it's better to have a metric config and this be the default value for it.

Reply:
Done 🫡